arduino/opencr_arduino/opencr/libraries/turtlebot3/turtlebot3_waffle/turtlebot3_core/turtlebot3_core.ino

```diff
@@ -187,7 +224,8 @@ void setup()
 void loop()
 {
   receiveRemoteControlData();
-
+  uint32_t t = millis();
+  update_time();
```

Comment:
And add a comment here why this is here and why it's important. Your description to me in person would be valuable to capture.